frontend/src/pages/devices/List.tsx

```diff
@@ -78,23 +79,23 @@ export const List = () => {
                 <tbody>
                     {entities.map(entity =>
                         <tr key={ entity.id }>
-                            <th scope="row">
+                            <th className="align-middle">
```

Comment:
Looks like there are some other changes in here now, what do all of these do? It might be good to include some before/after screenshots. 

It's generally frowned upon to unrelated changes to an existing PR :P